admin/app/components/certification-centers/information-edit.hbs

```diff
@@ -51,10 +51,65 @@
         @value={{this.form.externalId}}
       />
     </div>
+
+    <div class="form-field">
+      <label for="data-protection-officer-first-name" class="field__label">Prénom du DPO</label>
+      {{#if (v-get this.form "dataProtectionOfficerFirstName" "isInvalid")}}
+        <div class="form-field__error" aria-label="Message d'erreur du champ Prénom du DPO">
+          {{v-get this.form "dataProtectionOfficerFirstName" "message"}}
+        </div>
+      {{/if}}
+      <Input
+        id="data-protection-officer-first-name"
+        @type="text"
+        class={{if
+          (v-get this.form "dataProtectionOfficerFirstName" "isInvalid")
+          "form-control is-invalid"
+          "form-control"
+        }}
```

Comment:
**suggestion**
Une autre manière de faire pour s'éviter la répétition de "form-control" et insister juste sur "is-invalid" : 
```suggestion
        class="form-control {{if
          (v-get this.form "dataProtectionOfficerFirstName" "isInvalid")
          "is-invalid"
        }}"
```